datafusion/proto/src/logical_plan/from_proto.rs

```diff
@@ -1338,7 +1338,11 @@ pub fn parse_expr(
             in_list.negated,
         ))),
         ExprType::Wildcard(protobuf::Wildcard { qualifier }) => Ok(Expr::Wildcard {
-            qualifier: qualifier.clone(),
+            qualifier: if qualifier.is_empty() {
```

Comment:
I think this is fine -- I don't think an empty string `""` is a valid qualifier anyways so not being able to distinguish between `""` and `None` is not important